src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue

```diff
@@ -145,17 +145,15 @@ export default {
 		@update:open="handleCloseModal">
 		<template v-if="hasNavigation" #navigation="{ isCollapsed }">
 			<ul v-if="!isCollapsed"
-				:aria-label="settingsNavigationAriaLabel"
-				class="navigation-list"
-				role="tablist">
+				class="navigation-list">
 				<li v-for="section in sections" :key="section.id">
-					<a :aria-selected="section.id === selectedSection"
+					<a :aria-current="section.id === selectedSection"
```

Comment:
Choose your weapon

// no need in "false" actually
```suggestion
					<a :aria-current="section.id === selectedSection ? 'true' : undefined"
```

```suggestion
					<a :aria-current="section.id === selectedSection ? 'true' : 'false'"
```

```suggestion
					<a :aria-current="section.id === selectedSection ? 'location' : undefined"
```

```suggestion
					<a :aria-current="(section.id === selectedSection).toString()"
```

```suggestion
					<a :aria-current="`${section.id === selectedSection}`"
```